src/utils/qr.js

```diff
@@ -24,7 +24,7 @@ function generate(url) {
 function getImage({ order, orderId, walletAddress }) {
   const totalPrice = orderUtil.getTotalPrice(order)
 
-  const url = `${STATUS_UL}${SERVER_URL}/#/payment/${walletAddress}/${totalPrice}/${orderId}`
+  const url = `${STATUS_UL}${SERVER_URL}/payment/${walletAddress}/${totalPrice}/${orderId}`
```

Comment:
For this to work properly, we need to configure web-server to redirect all path's to '/' under the hood preserving the actual path in the browser.

e.g. When going to `app.ching.store/payment` web server can't figure out that we actually want to serve `app.ching.store/index.html`, tries to serve `app.ching.store/payment/index.html` which does not exist and returns 404

We were spared of this with `#` since web server ignores everything after `#` and always served root index.html, but with this setup, it's not going to work.

Here is a write up on how to configure it for single page react apps: https://legacy.gitbook.com/book/gkedge/react-router-in-the-real/details

Reply:
When I switched to Heroku I used some preset environment specifically for `create-react-app` so this might be covered by it and not an issue anymore. If it worked for you after the change it's likely the case - needs to be tested.

If so, search and replace `HashRouter` with `BrowserHistory`. That should be enough to get rid of hashes in all links.